app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -2065,43 +2065,36 @@ private void notifyAudioTrackUpdateToListeners() {
     }
 
     public void useVideoSource(final boolean videoEnabled) {
-        if (playQueue == null || isAudioOnly == !videoEnabled || audioPlayerSelected()) {
+        if (playQueue == null || audioPlayerSelected()) {
             return;
         }
 
         isAudioOnly = !videoEnabled;
 
-        // The current metadata may be null sometimes (for e.g. when using an unstable connection
-        // in livestreams) so we will be not able to execute the block below.
-        // Reload the play queue manager in this case, which is the behavior when we don't know the
-        // index of the video renderer or playQueueManagerReloadingNeeded returns true.
         getCurrentStreamInfo().ifPresentOrElse(info -> {
-            // In the case we don't know the source type, fallback to the one with video with audio
-            // or audio-only source.
+            // In the case we don't know the source type, fallback to the video with audio or
```

Comment:
```suggestion
            // In the case we don't know the source type, fallback to video with audio or
```

Reply:
I think you missed the line below your review comment (`// audio-only one`).

As I am talking about a specific source type, shouldn't I need to use `the` in this case?